process/transaction/shardProcess.go

```diff
@@ -1202,6 +1203,17 @@ func (txProc *txProcessor) makeSCRFromUserTx(
 		CallType:       vm.DirectCall,
 	}
 
+	isUserTxOfRelayedV3 := len(originalTx.InnerTransactions) > 0 &&
```

Comment:
does this happen also for relayed V1 and V2? could this fix also those? 
if so, I would remove the check isUserTxOfRelayedV3.

Reply:
pushed your suggestion, as it was only applying for v3 (I misunderstood)